src/pyfmi/fmi.pyx

```diff
@@ -1589,6 +1589,8 @@ cdef class FMUModelBase(ModelBase):
         nref = len(val_ref)
         cdef N.ndarray[FMIL.fmi1_real_t, ndim=1,mode='c'] val = N.array([0.0]*nref, dtype=float, ndmin=1)
 
+        if not nref: ## get_real([])
```

Comment:
As nref is of type "size_t" I think it is more clear to directly compare against 0. i.e:

```
if nref == 0:
   ...
```
It it was a general object, then I would agree with this syntax.

Reply:
Fixed.